CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) a
 
 ## dev
 
+### `Added`
```

Comment:
```suggestion
```
We already have an `Added` section just under, can you move the text in it?

Reply:
Got it